rands.c

```diff
@@ -234,12 +110,11 @@ int RandUniRange(const long first, const long last) {
 	}
 
 	r = l - f + 1;
-	return (long) (RandUni() * r) + f;
+	return (long) (RandUni(pcg_rng) * r) + f;

```

Comment:
Please call `pcg32_boundedrand_r`; see http://www.pcg-random.org/using-pcg-c.html#pcg32-boundedrand-r-rngptr-bound for an explanation.

Reply:
@dschlaep If I call pcg32_boundedrand_r() will I need to add an 
#ifdef RSOILWAT case to this function like RandUni()?